### PR TITLE
1.3.1 - New Dev Feature

### PR DESCRIPTION
**Pull request by hkatzdev**
Allow Slack to automate certain commands.

### CODE REVIEW

File: config/special_users.json [hunk 1]
```diff
@@ -7,5 +7,6 @@
     "U01230GJXS6",
     "UE8DH0UHM",
     "U015MCCBXBP",
-    "U011CFN98K1"
+    "U011CFN98K1",
+    "USLACKBOT"
```
Comment (lukec11) [inline config/special_users.json:11]: Adding slackbot has the potential to cause abuse - we shouldn't let that happen if we can avoid it. Please remove this item from the list.

File: config/special_users.json [hunk 2]
```diff
@@ -7,5 +7,6 @@
     "U01230GJXS6",
     "UE8DH0UHM",
     "U015MCCBXBP",
-    "U011CFN98K1"
+    "U011CFN98K1",
```
Comment (lukec11) [inline config/special_users.json:10]: Remove the comma for properly formatted JSON

**Review by lukec11 — CHANGES_REQUESTED** [review]
Just a few minor changes and this should be good to merge!

File: config/special_users.json (hunk 1 above)
Comment (hkatzdev) [reply config/special_users.json:11]: request
I was requested by a dev to add slackbot to the list.

File: config/special_users.json (hunk 2 above)
Comment (hkatzdev) [reply config/special_users.json:10]: Resolving as the comma is needed for slackbot.

File: config/special_users.json (hunk 1 above)
Comment (lukec11) [reply config/special_users.json:11]: Sorry, that's clearly not slack, much less our official slack. Would you be willing to provide a direct screenshot of the file within slack, so we know it isn't falsified? Perhaps a link would help as well.

File: config/special_users.json (hunk 2 above)
Comment (lukec11) [reply config/special_users.json:10]: Please see comment above.

File: config/special_users.json (hunk 1 above)
Comment (hkatzdev) [reply config/special_users.json:11]: slack
OFC - I am using a third party client called RipCord in order to speed slack up for me. While I am not allowed to post the link due to it being a private channel in a private slack (and quite frankly as a fellow dev you know this), I did take a screenshot. Please excuse the light mode - I do not use the slack web app often.

File: config/special_users.json (hunk 2 above)
Comment (hkatzdev) [reply config/special_users.json:10]: Please do not unresolve this comment again as it is needed for the above.

File: config/special_users.json (hunk 1 above)
Comment (lukec11) [reply config/special_users.json:11]: As the dev who wrote the above message, I am well aware of what it said - your modifications here are frankly not helping the situation. As much as we appreciate these PRs there doesn't seem to be a super valid reason for it. Closing until we can find said reason.

**Comment by hkatzdev** [comment]
Unlocking for spam and relocking for too-heated since this is a valid issue but it appears there is disagreement on who said what.

File: config/special_users.json (hunk 1 above)
Comment (hkatzdev) [reply config/special_users.json:11]: Quite frankly I am astonished that you are claiming there are modifications - please feel free to DM me in the slack about this. For now I agree that it is no longer productive to comment on this issue.

**Pull request closed without merging**